src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -24,7 +24,11 @@
 					:placeholder="$t('calendar', 'Calendar name …')"
 					@input="calendarNameChanged = true">
 			</div>
-
+			<template v-if="canBeShared">
+				<NcCheckboxRadioSwitch :checked.sync="isTransparent">
+					{{ $t('calendar', 'All events in this calendar show you as busy by default') }}
```

Comment:
"Events in this calendar show you as busy if the events are marked busy."

I thinkt the current wording is misleading. This is not about defaults. It's about whether the calendar is checked for business at all. Events with status free still show you as free.

Reply:
Or we could switch the logic (using `!isTransparent`) and use "Never show me as busy" since for "opaque" you can still set the VEVENT as FREE but for transparent, a BUSY will never be used.

Reply:
That works, too :+1: